Library/Homebrew/cask/cask_loader.rb

```diff
@@ -384,7 +384,8 @@ def load(config:)
           end
 
           if (disable_date = json_cask[:disable_date].presence)
-            reason = DeprecateDisable.to_reason_string_or_symbol json_cask[:disable_reason], type: :cask
+            disable_reason = json_cask[:disable_reason] || json_cask[:deprecation_reason]
```

Comment:
```suggestion
            disable_reason = json_cask.fetch(:disable_reason,  json_cask[:deprecation_reason])
```
or, if you want to handle empty strings:

```suggestion
            disable_reason = json_cask[:disable_reason].presence || json_cask[:deprecation_reason]
```